workers/unity/Packages/com.improbable.gdk.core/Config/ReceptionistConfig.cs

```diff
@@ -48,5 +48,16 @@ public static ReceptionistConfig CreateConnectionConfigForPhysicalAndroid(string
                 UseExternalIp = true
             };
         }
+
+        public static ReceptionistConfig CreateConnectionConfigForiOS()
+        {
+            return new ReceptionistConfig
+            {
+                // By default this config only connects to local IP (127.0.0.1)
+                // Replace this code with hardcoded IP to coneect to non-local deployment
+                ReceptionistHost = RuntimeConfigDefaults.ReceptionistHost,
```

Comment:
actually, I think we can remove this line because it will be set to it by default anyway. sorry about that :)
we only need the `UseExternalIp`

Reply:
Make sense, didn't think about the default config. Thanks for flagging